lib.php

```diff
@@ -1113,13 +1072,12 @@ function block_coursefeedback_parse_dates($datesraw) {
             // Wir speichern den jeweiligen (begin und end) timestamp für das laufende Jahr
             //ohne Rücksicht auf Jahreswechel überlappende Zeiträume
             $result[] = array(
-                'begin' => mktime(0, 0, 0, $begmonth, $begday, date("Y")),
-                'end' => mktime(0, 0, 0, $endmonth, $endday, date("Y"))
+                'begin' => gmmktime(0, 0, 1, $begmonth, $begday, date("Y")),
+                'end' => gmmktime(0, 0, 1, $endmonth, $endday, date("Y"))
```

Comment:
Wäre es nicht cooler, wenn hier die Zeitzone der Moodle-Seite verwendet wird? Sonst erscheint die Benachrichtung bei Moodle-Seiten in Neuseeland tagsüber. Müsste damit gehen: https://docs.moodle.org/dev/Time_API

Reply:
Wäre cooler ja aber damit müssen die Kiwis jetzt erstmal leben :)

Reply:
Auch auf die Liste.